### PR TITLE
Ticket5

### PR DESCRIPTION
**Pull request by RossLockhart**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -18,12 +20,20 @@ function App() {
         />
       </head>
       <div className="App">
-        <header className="App-header"></header>
+        <header className="App-header">
+          <h2>The Table-Top Appraisal Shop&trade;</h2>
```
Comment (mallionaire) [inline src/App.js:15]: Such fine marketing! 😄

File: src/Components/AllReviews.jsx [hunk 2]
```diff
@@ -1,27 +1,27 @@
...
   }, []);
 
   return (
     <section className="AllReviews">
-      <h1>All Reviews</h1>
+      <h1>Select a game to see reviews</h1>
       <ul>
         {reviews.map((review) => {
-          return <SingleReview review={review} />;
+          console.log(11111, review);
```
Comment (mallionaire) [inline src/Components/AllReviews.jsx]: Ahem... what is this 🙄

File: src/Components/CategoriesPage.jsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
+import { useEffect, useState } from "react";
+import { Link } from "react-router-dom";
+import "../css/CatergoriesPage.css";
+
+const CategoriesPage = () => {
+  const [categories, selectCategories] = useState([]);
+  useEffect(() => {
+    fetch("https://fe-games.herokuapp.com/api/categories")
```
Comment (mallionaire) [inline src/Components/CategoriesPage.jsx:8]: Would really recommend a refactor to axios, and extracting to an utils/api.js

File: src/Components/ReviewCard.jsx [hunk 4]
```diff
@@ -1,12 +1,17 @@
-function SingleReview(props) {
+import { Link } from "react-router-dom";
+
+function ReviewCard(props) {
   const review = props.review;
+  console.log(22222, review);
```
Comment (mallionaire) [inline src/Components/ReviewCard.jsx:5]: Not another one...

File: src/Components/ReviewPage.jsx [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+const ReviewPage = () => {
+  const [review, setReview] = useState([]);
+  const { review_id } = useParams();
+
+  useEffect(() => {
+    fetchReviewCard(review_id).then((reviewData) => {
+      setReview(reviewData);
+    });
+  }, [review_id]);
+  console.log(666666, review);
```
Comment (mallionaire) [inline src/Components/ReviewPage.jsx]: Don't make me say it again Ross!

File: src/Components/SingleCategory.jsx [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+    <section className="SingleCategory">
+      <h2>Your category is served.....{category_slug}</h2>
+
+      <ul>
+        {category.map((review) => {
+          return (
+            <li key={review.category}>
+              <b>
+                <p>
+                  <u>{review.category}</u>
```
Comment (mallionaire) [inline src/Components/SingleCategory.jsx]: I don;t think this is needed for each item, a little jarring for the user to see it over and over.

File: src/utils/api.js [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+  baseURL: "https://fe-games.herokuapp.com/api",
+});
+
+export const fetchCategorySlug = (category_slug) => {
+  return myApi.get(`/reviews?category=${category_slug}`).then((res) => {
+    return res.data.reviews;
+  });
+};
+
+export const fetchReviewCard = (review_id) => {
```
Comment (mallionaire) [inline src/utils/api.js]: Ditto above, this fetches a singleReview, not a reviewCard, you're displaying the info on a review card

File: src/utils/api.js [hunk 8]
```diff
@@ -0,0 +1,17 @@
+import axios from "axios";
```
Comment (mallionaire) [inline src/utils/api.js:1]: Please use these functions 👍

File: src/Components/SingleCategory.jsx [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+              <p>
+                <u>{review.title}</u>
+              </p>
+              <p>
+                <b>Review:</b> {review.review_body}
+              </p>
+              <img
+                src={review.review_img_url}
+                width="250"
+                height="200"
```
Comment (mallionaire) [inline src/Components/SingleCategory.jsx]: Personally I'd extract the CSS, your components will bloat very quickly otherwise.

**Review by mallionaire — CHANGES_REQUESTED** [review]
Really great start, there are a few comments dotted about for you to think about. 

You'll also have to think about what you'd like to show your user if they tried to change the url to a category that doesn't exist, or have any reviews associated with it.

File: src/Components/CategoriesPage.jsx [hunk 10]
```diff
@@ -0,0 +1,39 @@
...
+      .then((res) => res.json())
+      .then((data) => {
+        selectCategories(data.categories);
+      });
```
Comment (emilyb93) [inline src/Components/CategoriesPage.jsx:12]: Lets extract this into an axios function in your api like we have with your others

**Review by emilyb93 — APPROVED** [review]
Extract your function from CategoriesPage to the api.js file, remove ANY console.logs floating about and this is good to go

**Pull request merged**